tests/src/system/basic/WskSequenceTests.scala

```diff
@@ -431,7 +431,7 @@ class WskSequenceTests
                 (trigger, name) => trigger.create(name, parameters = triggerPayload)
             }
             // action
-            val file = TestUtils.getTestActionFilename(s"$actionName.js")
+            val file = TestUtils.getTestActionFilename("echo.js")
```

Comment:
No, there are no changes to echo.
I'd prefer to change the name in both tests. Otherwise, we have some new tests some day, and they are also called echo. Then we have the same clache again with these tests.